src/libraries/System.Console/src/System/ConsolePal.Unix.cs

```diff
@@ -1513,5 +1514,36 @@ private static void HandleBreakEvent(object? state)
                 }
             }
         }
+
+        public sealed class ReadOnlyMemoryContentComparer : IEqualityComparer<ReadOnlyMemory<char>>
```

Comment:
```suggestion
        private sealed class ReadOnlyMemoryContentComparer : IEqualityComparer<ReadOnlyMemory<char>>
```

Reply:
Whoops, forgot to change the modifier when copied it from a benchmark project.